### PR TITLE
io_lib with foss

### PR DESCRIPTION
**Pull request by RvDijk**

### CODE REVIEW

File: easybuild/easyconfigs/i/io_lib/io_lib-1.14.8-foss-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ into memory. It has been compiled and tested on a variety of unix systems, MacOS X and MS Windows. 
+"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = [('http://sourceforge.net/projects/staden/files/%(namelower)s/%(version)s', 'download')]
+sources = [SOURCELOWER_TAR_GZ]
+
+files_to_copy = [
+    "bin", "lib", "include", "share"]
```
Comment (boegel) [inline easybuild/easyconfigs/i/io_lib/io_lib-1.14.8-foss-2016a.eb]: this is not used, it's only relevant for `MakeCp`, so please remove?

File: easybuild/easyconfigs/i/io_lib/io_lib-1.14.8-foss-2016a.eb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+sanity_check_paths = {
+    'files': ["bin/scramble", "bin/append_sff", "bin/ztr_dump"],
+    'dirs': [],
+}
+
+
```
Comment (boegel) [inline easybuild/easyconfigs/i/io_lib/io_lib-1.14.8-foss-2016a.eb]: please remove duplicate empty line

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2547.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/4a5c231674f189d21684aa85db198eac for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2104.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/a7f53bb95a6bd6389d06de6b661ead22 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @RvDijk!

File: easybuild/easyconfigs/i/io_lib/io_lib-1.14.8-foss-2016a.eb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+source_urls = [('http://sourceforge.net/projects/staden/files/%(namelower)s/%(version)s', 'download')]
+sources = [SOURCELOWER_TAR_GZ]
+
+sanity_check_paths = {
+    'files': ["bin/scramble", "bin/append_sff", "bin/ztr_dump"],
+    'dirs': [],
+}
+
+moduleclass = 'bio'
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/i/io_lib/io_lib-1.14.8-foss-2016a.eb]: eh, is everything 'bio' nowadays?! :)
although this is really a fork of code out of `staden` bioinfo package, I'd suggest to reconsider the category...

**Comment by fgeorgatos** [comment]
@boegel: reconsider if category makes sense

**Comment by boegel** [comment]
@fgeorgatos suggestions? `lib`?

**Comment by pescobar** [comment]
thanks for the easyconfig.

As a side note, this one is missing a lot of dependencies like bzip2, zlib, curl, 

```
[easybuild@login11 easyconfigs]$ ldd ~/test-installs/apps/io_lib/1.14.8-goolf-1.7.20/bin/srf2fastq
	linux-vdso.so.1 =>  (0x00007f5106f28000)
	libstaden-read.so.11 => /scicore/home/scicore/easybuild/test-installs/apps/io_lib/1.14.8-goolf-1.7.20/lib/libstaden-read.so.11 (0x00007f5106c67000)
	libbz2.so.1 => /lib64/libbz2.so.1 (0x0000003feb200000)
	libm.so.6 => /lib64/libm.so.6 (0x0000003e87e00000)
	libpthread.so.0 => /lib64/libpthread.so.0 (0x0000003e87600000)
	libc.so.6 => /lib64/libc.so.6 (0x0000003e87200000)
	libz.so.1 => /lib64/libz.so.1 (0x0000003e88200000)
	libcurl.so.4 => /usr/lib64/libcurl.so.4 (0x00007f51069fc000)
	/lib64/ld-linux-x86-64.so.2 (0x0000003e86a00000)
	libidn.so.11 => /lib64/libidn.so.11 (0x0000003e8e200000)
	libldap-2.4.so.2 => /lib64/libldap-2.4.so.2 (0x0000003fe9e00000)
	librt.so.1 => /lib64/librt.so.1 (0x0000003e87a00000)
	libgssapi_krb5.so.2 => /lib64/libgssapi_krb5.so.2 (0x0000003fe9200000)
	libkrb5.so.3 => /lib64/libkrb5.so.3 (0x0000003fe6a00000)
	libk5crypto.so.3 => /lib64/libk5crypto.so.3 (0x0000003e8ca00000)
	libcom_err.so.2 => /lib64/libcom_err.so.2 (0x0000003fe6e00000)
	libssl3.so => /usr/lib64/libssl3.so (0x0000003fe7a00000)
	libsmime3.so => /usr/lib64/libsmime3.so (0x0000003fea200000)
	libnss3.so => /usr/lib64/libnss3.so (0x0000003fe7200000)
	libnssutil3.so => /usr/lib64/libnssutil3.so (0x0000003fe7600000)
	libplds4.so => /lib64/libplds4.so (0x0000003fe7e00000)
	libplc4.so => /lib64/libplc4.so (0x0000003fe8600000)
	libnspr4.so => /lib64/libnspr4.so (0x0000003e8b600000)
	libdl.so.2 => /lib64/libdl.so.2 (0x0000003e86e00000)
	libssh2.so.1 => /usr/lib64/libssh2.so.1 (0x0000003febc00000)
	liblber-2.4.so.2 => /lib64/liblber-2.4.so.2 (0x0000003fe8e00000)
	libresolv.so.2 => /lib64/libresolv.so.2 (0x0000003e89a00000)
	libsasl2.so.2 => /usr/lib64/libsasl2.so.2 (0x0000003e8da00000)
	libkrb5support.so.0 => /lib64/libkrb5support.so.0 (0x0000003e8ce00000)
	libkeyutils.so.1 => /lib64/libkeyutils.so.1 (0x0000003e8d600000)
	libssl.so.10 => /usr/lib64/libssl.so.10 (0x0000003fea600000)
	libcrypto.so.10 => /usr/lib64/libcrypto.so.10 (0x0000003e8ba00000)
	libcrypt.so.1 => /lib64/libcrypt.so.1 (0x0000003e8a200000)
	libselinux.so.1 => /lib64/libselinux.so.1 (0x0000003e88600000)
	libfreebl3.so => /lib64/libfreebl3.so (0x0000003e89e00000)
```

**Pull request merged**